README.md

```diff
@@ -168,6 +169,11 @@ remove the line in configure that sets 'fix_srcfile_path' to a 'cygpath'
 command.  ('cygpath' is not present in MingW, and is not required when
 using MingW-style paths.)
 
+To build static library for ARM64 with MSVC using visual studio solution, msvc_build folder have
```

Comment:
Is this really required?  Can't it be built like the other Windows ports?

Reply:
as there is no Linux build environment support in Window 10 on ARM, so VS solution is the only approach,

Reply:
We should be able to use the "cl on msys2" way like the windows arm32 port does.

Reply:
Also, clang Windows 10 ARM64 targets are ready to use, although it can be supported later.